system/doc/reference_manual/data_types.xml

```diff
@@ -87,6 +87,19 @@
 11> <input>1_234.333_333</input>
 1234.333333
 </pre>
+<marker id="numeric_comparisons" />
+    <section>
+      <title>Comparisons</title>
+      <p>Both integers and floats share the same linear order. That is,
+        <c>1</c> compares less than <c>2.4</c>, <c>3</c> compares greater than
+        <c>2.99999</c>, and <c>5</c> is equal to <c>5.0</c>.</p>
+      <p>When wanting to match a specific number without also matching its
+        integer- or floating-point twin, the term equivalence operators
+        (<c>=:=</c>, <c>=/=</c>) should be used instead.</p>
+      <p>Note that since OTP 27, <c>0.0</c> and <c>-0.0</c> are <em>not</em>
+        considered to be the same term since they can be distinguished through
+        the use of string or bitstring conversion functions.</p>
```

Comment:
At the end of the next section it is stated that we don not follow IEEE completely, and that Nan, +Inf and -Inf are not implemented in Erlang.  I suggest writing there that 0.0 vs. -0.0 *is* since it is also an IEEE feature, but one that we *do* support.